index.html

```diff
@@ -49,16 +49,17 @@
 
         <button>Donate Now</button>
 
-    </div>
+    </header>
 
     <div class="content">
```

Comment:
Consider using different [semantic tags](https://www.w3schools.com/html/html5_semantic_elements.asp) across the page. Fallback to `div`s when absolutely necessary - quite often there is a better alternative.